app.js

```diff
@@ -337,6 +349,28 @@ function getList( type ) {
   return list;
 }
 
+/**
+ * Add demo tool object
+ *
+ * @param  {Array}   toolList        List of all tools
+ * @param  {Boolean} isDemoToolAdded Indicator for existing demo tool
+ * @return {Array}                   Updated tool list
+ */
+function addDemoTool( toolList, isDemoToolAdded ) {
+  if ( !isDemoToolAdded ) {
+    demoTool.tags   = [];
+
+    _.each( config.platforms, function( platform ) {
+        demoTool[ platform.name ] = {};
+        demoTool.stars[ platform.name ] = 10000;
+        demoTool.tags = [ 'images', 'css', 'perf', '60fps' ];
```

Comment:
Yes, I agree on that. This info should be part of the object initialisation as it won't change during the process. I will change it, right away.